src/osd/osd_types.h

```diff
@@ -66,17 +66,24 @@
 #define CEPH_OSD_FEATURE_INCOMPAT_FASTINFO CompatSet::Feature(15, "fastinfo pg attr")
 
 
-/// max recovery priority for MBackfillReserve
-#define OSD_RECOVERY_PRIORITY_MAX 255u
+/// min recovery priority for MBackfillReserve
+#define OSD_RECOVERY_PRIORITY_MIN 0
 
-/// base recovery priority for MBackfillReserve
-#define OSD_RECOVERY_PRIORITY_BASE 230u
+/// base backfill priority for MBackfillReserve
+#define OSD_BACKFILL_PRIORITY_BASE 100
 
 /// base backfill priority for MBackfillReserve (degraded PG)
-#define OSD_BACKFILL_DEGRADED_PRIORITY_BASE 200u
+#define OSD_BACKFILL_DEGRADED_PRIORITY_BASE 140
```

Comment:
Hmm, this would be important only if we compare priorities across different OSDs, right? I couldn't find anything in the code that would send this over the network, but maybe I'm not yet that familiar with it ;)

How I was thinking about this: priorities will be consistent within one process and restarting one should also reset all reservations on remote ends so there should be no situation that one OSD would present inconsistent priorities. Only comparing them across different OSDs could go out of control.